README.md

```diff
@@ -77,6 +77,21 @@ yourlogger.addHandler(airbrake.AirbrakeHandler())
 ```
 _by default, the `AirbrakeHandler` only handles logs level ERROR (40) and above_
 
+#### Additional Options
+More options are available to configure this library. Similar to setup, you can do this with environment variables
+```
+export AIRBRAKE_HOSTNAME=sassbox-101.prod.api
```

Comment:
That's a much better idea.
![tumblr_mu02erjkia1rk6p9mo1_500](https://cloud.githubusercontent.com/assets/1009156/23083905/fa251966-f52d-11e6-844e-733c938a9c6e.gif)